packages/expo-module-template/android/build.gradle

```diff
@@ -17,6 +6,21 @@ apply plugin: 'maven'
 group = 'host.exp.exponent'
 version = '8.1.0'
 
+buildscript {
```

Comment:
Will this create `buildscript` sections in new modules based on this template?

Reply:
Yes, because new modules are already being written in Kotlin (there is an implementation below) and I think we still don't want to assume that people using our modules have Kotlin configured in their projects, so until then we should have this `buildscript` there and download Kotlin's gradle plugin if it's not there yet. Anyway, we will use `ext.kotlinVersion` so users will be able to set one specific version and reduce number of downloads.